Makefile

```diff
@@ -4,14 +4,57 @@ ifdef JENKINS_URL
 	MAKE_INCLUDES := .
 else
 	# For local dev, use the installed vivarium_build_utils package
-	MAKE_INCLUDES := $(shell python -c "from vivarium_build_utils.resources import get_makefiles_path; print(get_makefiles_path())")
+	MAKE_INCLUDES := $(shell python -c "import vivarium_build_utils.resources" &>/dev/null && python -c "from vivarium_build_utils.resources import get_makefiles_path; print(get_makefiles_path())")
 endif
 
-PACKAGE_NAME = layered_config_tree
+PACKAGE_NAME = $(notdir $(CURDIR))
 
 # Include makefiles from vivarium_build_utils
+ifneq ($(MAKE_INCLUDES),) # if not an empty string
 include $(MAKE_INCLUDES)/base.mk
 include $(MAKE_INCLUDES)/test.mk
+endif
+
+ifeq ($(MAKE_INCLUDES),) # empty string
+help:
+	@echo
+	@echo "For Make's standard help, run 'make --help'."
+	@echo
+	@echo "Most of our Makefile targets are provided by the vivarium_build_utils"
+	@echo "package. To access them, you need to create a development environment first."
+	@echo
+	@echo "make build-env"
+	@echo
+	@echo "USAGE:"
+	@echo "  make build-env name=<environment_name> [py=<python_version>]"
+	@echo
+	@echo "ARGUMENTS:"
+	@echo "  name  [required]  Name of the conda environment to create"
+	@echo "  py    [optional]  Python version (defaults to latest supported)"
+	@echo
+	@echo "EXAMPLE:"
+	@echo "  make build-env name=vivarium_dev"
+	@echo "  make build-env name=vivarium_dev py=3.9"
+	@echo
+	@echo "After creating the environment:"
+	@echo "  1. Activate it: 'conda activate <environment_name>'"
+	@echo "  2. Run 'make help' again to see all newly available targets"
+	@echo
+endif
 
-install:
```

Comment:
I really can't think of a reason why this should be defined here instead of the version in vbu